docs/configure/environment.md

```diff
@@ -1,4 +1,4 @@
-### Configuration Variables
+### Environment Variables
```

Comment:
can you add please CLUSTER_NAME, since we start using it for tagging

Reply:
fixed